CRM/Financial/BAO/Order.php

```diff
@@ -742,17 +742,20 @@ public function getLineItem($index): array {
    */
   protected function fillMembershipLine(array $lineItem): array {
     $fields = $this->getPriceFieldsMetadata();
-    $field = reset($fields);
```

Comment:
This line is the key to this change - it assumed there was only one field in the prices set - however, there is one field per membership_organization (in the membership_type table)